conan/recipe/all/conandata.yml

```diff
@@ -0,0 +1,5 @@
+sources:
+  "master":
```

Comment:
We need to create a tag after this PR to ensure a stable hash

Reply:
I will update it once we merge this